src/components/step/YouTubeStep/YouTubeStep.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import StepLayout from '../StepLayout'
+import StepTitle from '../StepTitle';
+import StepText from '../StepText';
+import StepDetailText from '../StepDetailText'
+import StepNextButton from '../StepNextButton';
+
+export interface YouTubeStepProps {
+    properties: { [key: string]: any };
```

Comment:
I'd have the step take explicit properties.  

e.g. videoId, height, title, text, transcript, and nextButtonText as explicit things here...that makes storybook testing very easy and also potential re-use of this component easy

Reply:
(Don't feel super strongly about it though)

Reply:
That does make it more nicely self-documenting so I refactored these out. Styles are even more opaque and in need of documentation...so I started playing with a couple of approaches and may refactor how we override those in a follow on PR.